WordPress/src/main/res/values/strings.xml

```diff
@@ -433,6 +433,7 @@
     <string name="stats_no_blog">Stats couldn\'t be loaded for the required blog</string>
     <string name="stats_generic_error">Required Stats couldn\'t be loaded</string>
     <string name="stats_sign_in_jetpack_different_com_account">To view your stats, sign in to the WordPress.com account you used to connect Jetpack.</string>
+    <string name="stats_enable_rest_api_in_jetpack">To view your stats, enable the REST API module in Jetpack.</string>
```

Comment:
Right, will update the error string. 
I had used, more or less, the wording used in the Jetpack error: (API calls to this blog have been disabled.)